src/main/java/pages/WelcomePage.java

```diff
@@ -0,0 +1,34 @@
+package pages;
+
+import io.appium.java_client.MobileElement;
+import io.appium.java_client.pagefactory.AndroidFindBy;
+import io.appium.java_client.pagefactory.iOSXCUITFindBy;
+import utils.MobileManagement;
+
+public class WelcomePage extends BasePage {
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(xpath = "//android.view.ViewGroup[@content-desc=\"continueAsGuest\"]/android.widget.TextView")
+    private MobileElement continueAsGuest;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "logIn")
+    private MobileElement btnLogin;
+
+    @iOSXCUITFindBy(xpath = "")
+    @AndroidFindBy(accessibility = "signUp")
+    private MobileElement btnSignUp;
+
+    public void clickLogin() {
+        btnLogin.click();
+    }
+
+    public void clickSignUp() {
+        btnSignUp.click();
+    }
+
+    public boolean isOpenApp() {
+        return MobileManagement.getDriver().isAppInstalled("com.azhavrid.movieswiperr");
+    }
```

Comment:
this methos does not go here because it is not a method from the Welcome page, it is a method from the app in general(if you call it outside this page it will also work and respond correctly). This could go in a helpers class

Reply:
👍 
I will move the method for MobileManagement class